spec/02-identifiers-names-and-scopes.md

```diff
@@ -16,9 +16,10 @@ which are collectively called _bindings_.
 
 Bindings of different kinds have precedence defined on them:
 
-1. Definitions and declarations that are local, inherited, or made
+1. Local definitions and declarations have the highest precedence.
+1. Definitions and declarations that are either inherited, or made
    available by a package clause and also defined in the same compilation unit
-   as the reference to them, have the highest precedence.
+   as the reference to them, have the next highest precedence.
```

Comment:
This spec change is 3.0+ only? I'd like the spec in this repo to keep reflecting what 2.13 does. But adding information in place about differences in (-Xsource:)3.0+ would be great.

cc @Sporarum - if you have ideas how to best handle this, let us know.

Reply:
maybe just a suffix clause "...when compiling for Scala 3; otherwise, legacy Scala 2 precedence is highest precedence."

Reply:
Now I suspect the change should not be under `-Xsource`, but a separate flag to retain the legacy (wrong) precedence. The code affected by this spec tweak is either wrong (a bug) or can be fixed easily and portably.